doc/examples/widgets/lib/main.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:flame/flame.dart';
+import 'package:dashbook/dashbook.dart';
+
+import 'package:flame/widgets/nine_tile_box.dart';
+
+void main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+
+  final nineTileBoxImage = await Flame.images.load('nine_tile_box.png');
+  final dashbook = Dashbook();
+
+  dashbook.storiesOf('NineTileBox').decorator(CenterDecorator()).add(
+      'default',
+      (ctx) => Container(
+            width: ctx.numberProperty('width', 200),
+            height: ctx.numberProperty('height', 200),
+            child: NineTileBox(
+              image: nineTileBoxImage,
+              tileSize: 16,
+              destTileSize: 50,
+              child: const Center(
+                  child: const Text('Cool label',
```

Comment:
please format like this:
```
    child: Foo(
         a: b,
         cccccc,
        // ...
    ),
```

Reply:
this is the result of the `flutter format` command, I think we should keep to it, so at least we have the same formatting base, even though I prefer the style suggested by you.

Reply:
it only tells you to format like that because you are missing a trailing comma. my formatting suggestion is formatter valid :)

Reply:
fair enough, I didn't knew about the trailing comma thing, but my desire for this kind of thing be on our lint persists, it waster a lot of our time this not been there, as is very hard to remember all these stuff, the trailing coma should have been on the formatter, do you have any idea on how to set it up @luanpotter ?